fastlane/metadata/en-US/release_notes.txt

```diff
@@ -1,12 +1,4 @@
-We updated Comment Notifications with a spiffy new look and feel. (You might recognize it from My Site Comments!) The Quick Start design in your site menu and dashboard is rocking a new look, too.
+* [*] Enhances the exit animation of notices. [#18182]
+* [*] Media Permissions: display error message when using camera to capture photos and media permission not given [https://github.com/wordpress-mobile/WordPress-iOS/pull/18139]
+* [***] My Site: your My Site screen now has two tabs, "Site Menu" and "Home". Under "Home", you'll find contextual cards with some highlights of whats going on with your site. Check your drafts or scheduled posts, your today's stats or go directly to another section of the app. [#18240]
```

Comment:
I was surprised by this diff, because usually this file only changes with the editorialized release notes.

What happened here is not a mistake, but a byproduct of the new localization tooling and the timing at which I created this beta.

The new tooling copies `WordPress/Resources/release_notes.txt` into this file. Previously we used the `en-GB` locale to get this information. IMO the current approach is an improvement.

https://github.com/wordpress-mobile/WordPress-iOS/blob/00acc5f188bf11e18b7286e9a8bc931450d92580/fastlane/lanes/localization.rb#L234-L237

The only side effect is that, if you create a beta before adding the editorialized release notes, this fill will be updated with the raw ones. That's exactly what happened in this PR.

Note that this PR also includes the editorialized release notes. But, those were added after running the `new_beta_release` lane. I run the beta before porting the release notes because I wanted to make the most of the "dead" time in which CI made the build.

I thought of running the lane to update this file, but decided not to because it will run when we'll make the next beta.

_The same observations apply to Jetpack, but with different paths to the `.txt` files involved._

Reply:
> The only side effect [of the new tooling copying the `release_notes.txt` file from `Resources/` is that, if you create a beta before adding the editorialized release notes, this fill will be updated with the raw ones. That's exactly what happened in this PR.

We also have a note to remove this copying mechanism. I think it will make things clearer. Or, at least, more deterministic, in that `fastlane/metadata/en-US/release_notes.txt` will always change to first include the raw release notes and then the editorialized ones as part of the code freeze and release cycle.

https://github.com/wordpress-mobile/WordPress-iOS/blob/00acc5f188bf11e18b7286e9a8bc931450d92580/fastlane/lanes/localization.rb#L231-L232